bitnami/thanos/templates/bucketweb/service.yaml

```diff
@@ -28,6 +28,11 @@ spec:
       {{- else if eq .Values.bucketweb.service.type "ClusterIP" }}
       nodePort: null
       {{- end }}
-  selector: {{- include "thanos.matchLabels" . | nindent 4 }}
+  selector:
+    {{- if .Values.bucketweb.service.labelSelectorsOverride }}
+    {{- include "thanos.tplValue" (dict "value" .Values.bucketweb.service.labelSelectorsOverride "context" $) | nindent 4 }}
+    {{- else }}
+    {{- include "thanos.matchLabels" . | nindent 4 }}
     app.kubernetes.io/component: bucketweb
```

Comment:
Shouldn't `app.kubernetes.io/component` label be kept even if the `labelSelectorsOverride` is provided?

Reply:
I added it also there. I am not really sure here - it's not a problem for my use case, but my initial thinking was that either someone wants to take the default chart selector or he or she might want to customize these entirely.

Reply:
Should I change the `labelSelectorsOverride` to something without Override like just `labelSelectors` because it doesn't really override all selectors?

Reply:
> my initial thinking was that either someone wants to take the default chart selector or he or she might want to customize these entirely.

You are right @janlud.

Reply:
Oh, you changed it already! Sorry for the misunderstanding. After giving it a second thought, it makes more sense the first implementation that you proposed.

We are going to revert the change on your behalf and merge the PR.

Reply:
Yeah, I wanted to speed it up in case you insist on the change. All good! :) Thanks